### ISSUE TITLE
Select articles in the citations note and save their metadata

### ISSUE THREAD

**Issue by egonw** [zuphilip/zotero-open-citations#6]
I love the idea to have a note for an article with the articles that cite it. I guess the note should be updated at intervals (maybe a third issue :), but it would be great if citing articles from that note can be easily bookmarked in Zotero themselves, but clicking the DOI in that note created by this plugin.

**Comment by zuphilip** [comment]
Good idea! By clicking on the DOI or button next to the DOI could make a Crossref lookup (or some other translator) and then save the resulting metadata.

However, the possibilities within a note might be limited, because this is some special configured TinyMC editor which IMO does not allow any (?) interactive elements. It should be possible to make another XUL element for the selection.

A difficulty is currently also that the user will only see the DOIs and the creation date/year and then has to make a selection. The DOI does not help much for any decision and therefore there is only the date. That is the reason why I implemented to retrieve the last 3 resp. N citations automatically in version 0.2.0 in a special folder, because it look as the only reasonable choice. Did you try that out?
